### PR TITLE
fix install doc by add more desc for CUDA

### PR DESCRIPTION
**Pull request by JiabinYang**
This PR adds more desc for the relationship of  CUDA and cuDNN version with paddle

### CODE REVIEW

File: doc/fluid/beginners_guide/install/Tables.md [hunk 1]
```diff
@@ -222,21 +222,21 @@ PaddePaddle通过编译时指定路径来实现引用各种BLAS/CUDA/cuDNN库。
 		<td> 只支持CPU对应版本的PaddlePaddle，具体版本请参见<a href=https://pypi.org/project/paddlepaddle/#history>Pypi</a> </td>
 	</tr>
 	<tr>
-		<td> paddlepaddle-gpu==1.5.1 </td>
-		<td> 使用CUDA 9.0和cuDNN 7编译的1.5.1版本 </td>
+		<td> paddlepaddle-gpu==[版本号] 例如 paddlepaddle-gpu==1.5.1 </td>
+		<td> 默认的支持CUDA 10.0和cuDNN 7的对应PaddlePaddle版本的安装包 </td>
```
Comment (lanxianghit) [inline doc/fluid/beginners_guide/install/Tables.md]: “默认的支持” 感觉语句不通顺，想表达的意思是不加版本号的情况下，默认安装包支持xxx？建议稍微修改一下

File: doc/fluid/beginners_guide/install/Tables.md [hunk 2]
```diff
@@ -222,21 +222,21 @@ PaddePaddle通过编译时指定路径来实现引用各种BLAS/CUDA/cuDNN库。
...
    </tbody>
 </table>
 </p>
 
 您可以在 [Release History](https://pypi.org/project/paddlepaddle-gpu/#history) 中找到PaddlePaddle-gpu的各个发行版本。
+> 其中`postXX` 对应的是CUDA和cuDNN的版本，`postXX`之前的数字代表Paddle的版本
 
-需要注意的是，<code> paddlepaddle-gpu </code> 命令在windows环境下，会默认安装CUDA 8.0和cuDNN 7编译的PaddlePaddle
-
+需要注意的是，<code> paddlepaddle-gpu </code> 命令在windows环境下，会默认安装CUDA 9.0和cuDNN 7编译的PaddlePaddle
```
Comment (lanxianghit) [inline doc/fluid/beginners_guide/install/Tables.md]: paddle-gpu不是“命令”吧？

File: doc/fluid/beginners_guide/install/Tables.md (hunk 1 above)
Comment (JiabinYang) [reply doc/fluid/beginners_guide/install/Tables.md]: done

File: doc/fluid/beginners_guide/install/Tables.md (hunk 2 above)
Comment (JiabinYang) [reply doc/fluid/beginners_guide/install/Tables.md]: done

**Review by lanxianghit — APPROVED** [review]
LGTM

**Review by cyj1986 — APPROVED** [review]

**Pull request merged**